pf/tests/testdata/updateprogram/Pulumi.yaml

```diff
@@ -27,4 +27,4 @@ outputs:
   testOptionalString__expect: |-
     input2
   testSetOptionals__actual: ${set.setOptionals}
-  testSetOptionals__expect: [ "${r.requiredInputStringCopy}", a, b ]
```

Comment:
This worries me quite bit! After using the new algorithm the order changed in the test. Perhaps the values are after all reordered for sets in TF somehow?

Reply:
@iwahbe we actually assume order preservation correct?